cmp/form/field/BaseFieldModel.js

```diff
@@ -23,9 +23,6 @@ import {PendingTaskModel} from '@xh/hoist/utils/async';
  */
 export class BaseFieldModel extends HoistModel {
 
-    /** @member {FormModel} */
```

Comment:
FYI I find these annotations helpful to get typing within this containing class's code, and when looking for usages of any of the member class's properties during refactoring. (Noticed one or two of them went away in the TODO example app.)

I certainly don't think we need these on _every_ member - not by any means - but for Hoist models with non-trivial APIs, I often find they have value in a few ways while we are working on the Hoist codebase itself.

(I don't think that they overly imply the member to be public. We could even go further and add `@private` to the member jsdoc to make that more clear, vs. the leading underscore convention, which I've seen developers very happily ignore. But I have not been doing that - waiting for us to take the plunge to the new private fields syntax instead.... 😃)